backend-api/src/functions/services/events/tests/eventService.test.ts

```diff
@@ -274,7 +274,7 @@ describe("Event Service", () => {
           documentType: "NFC_PASSPORT",
           ipAddress: "mockIpAddress",
           txmaAuditEncoded: "mockTxmaAuditEncoded",
-          redirect_uri: undefined,
+          redirect_uri: "https://example.com/yay",
```

Comment:
```suggestion
          redirect_uri: "https://mockRedirectUri.com",
```

Apologies for the boring suggestion but typically we represent the mock data like above. I have nothing against yay myself but we tend not to use anything like that in this repo

Reply:
The same would then go for the assertion below 😞

Reply:
Of course!  I will comply :D